openapi.yaml

```diff
@@ -29228,6 +29228,7 @@ paths:
                                 description: The unique identifier for the core compute region where this image is stored.
                                 example: us-iad
                                 type: string
+                                x-linode-cli-display: 11
```

Comment:
Can you remove all the `x-linode-cli-display: 11`? It caused the output only display `region` instead of both `region` and `status`.